.github/workflows/firefox.yml

```diff
@@ -0,0 +1,136 @@
+name: Firefox
+on:
+  push:
+    branches: ["main"]
+    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
+  pull_request:
+    branches: ["main"]
+    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
```

Comment:
This is going to take an awfully long time to run.  Do we really want to hold pull requests up on this completing?

Reply:
Not sure. @mxinden suggested to make this an async CI step (see https://github.com/mozilla/neqo/pull/1834#issuecomment-2060532361), but I don't quite know how to do this in practice.

FWIW, merging isn't blocked by this, since we currently only require these four CI workflows to pass for a merge: https://github.com/mozilla/neqo/settings/branch_protection_rules/46171042